README.md

```diff
@@ -5,225 +5,3 @@
 
 A command line interface to interact with EdgeX microservices. Replaces the need to manually construct complex CURL commands and/or maintain developer scripts.
```

Comment:
I think we need a placeholder link to the on-line docs, which should cover how to install, run, etc.

Reply:
yes, this README needs to be updated in a separate PR once we have the on-line docs ready.